src/products/Product.java

```diff
@@ -0,0 +1,127 @@
+package products;
+
+import enums.KindOfProduct;
+import enums.Models;
+import enums.Producers;
+
+public class Product {
+
+    private KindOfProduct article;
+
+    public KindOfProduct getArticle() {
+        return article;
+    }
+
+    public void setArticle(KindOfProduct article) {
+        this.article = article;
+    }
+
+    private Producers producer;
+
+    public Producers getProducer() {
+        return producer;
+    }
+
+    public void setProducer(Producers producer) {
+        this.producer = producer;
+    }
+
+    private Models model;
+
+    public Models getModel() {
+        return model;
+    }
+
+    public void setModel(Models model) {
+        this.model = model;
+    }
+
+    private String id;
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    private String description;
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    private double display;
+
+    public void setDisplay(double display) {
+        this.display = display;
+    }
+
+    public double getDisplay() {
+        return display;
+    }
+
+    private double storage;
+
+    public void setStorage(double storage) {
+        this.storage = storage;
+    }
+
+    public double getStorage() {
+        return storage;
+    }
+
+    private double purchasePrice;
+
+    public void setPurchasePrice(double purchasePrice) {
+        this.purchasePrice = purchasePrice;
+    }
+
+    public double getPurchasePrice() {
+        return purchasePrice;
+    }
+
+    private double sellPrice;
+
+    public void setSellPrice(double sellPrice) {
+        this.sellPrice = sellPrice;
+    }
+
+    public double getSellPrice() {
+        return sellPrice;
+    }
+
```

Comment:
La formattazione intermezzando  il codice  della classe e i metodi mi risulta poco leggibile e non tanto in linea con quello che abbiamo fatto fino ad ora.  Non so se ci siano standard su questo, possiamo chiedere a Lorenzo. Però in ogni caso prima di fare cambiamenti di questo tipo mettiamoci d'accordo.

Reply:
Suggerisci di mettere i getter e i setter in "compartimenti" loro? Tipo: 
-variabile
-variabile

-set
-set

-get
-get?